### PR TITLE
Final release

### PR DESCRIPTION
**Pull request by Setareh-Neshatdoust**

### CODE REVIEW

File: code/client/README.md [hunk 1]
```diff
@@ -8,6 +8,45 @@ This document lists all components and routes that compose the Kiruna Explorer a
...
+    - [Nav Bar](#nav-bar)
+    - [User Context](#user-context)
+    - [Sidebar Provider](#sidebar-provider)
+    - [Toast Provider](#toast-provider)
+  - [2. Anonymous User - Map](#2-anonymous-user---map)
+    - [ExploreMap](#exploremap)
+  - [3. General Pages](#3-general-pages)
+    - [Home](#home)
+    - [Login](#login)
+  - [4. Urban Planner](#4-urban-planner)
```
Comment (SaraPoiatti) [inline code/client/README.md:22]: The directory UrbanPlanner/MapFeatures is missing

File: code/client/README.md [hunk 2]
```diff
@@ -60,82 +99,282 @@ export default tseslint.config({
...
+
+The `App` component defines a set of routes to handle navigation between different parts of the application.
+
+---
+
+### **Root Route (`/`)**
+- **Functionality**:  
+  When a user navigates to the root path (`/`), this route checks if the user is logged in:  
+  - **If logged in**: Redirects the user to the `/explore-map` route.  
+  - **If not logged in**: Redirects the user to the `/login` route.
```
Comment (SaraPoiatti) [inline code/client/README.md:112]: right now the path ('/') navigate to '/explore-map' for both the logged user and the guest user

File: code/client/README.md [hunk 3]
```diff
@@ -60,82 +99,282 @@ export default tseslint.config({
...
+- **Functionality**:  
+  Displays the `Home` component, which is a general public landing page.  
+  - This route does not require login and can be accessed by any user.
+
+---
+
+### **Explore Map Route (`/explore-map`)**
+- **Functionality**:  
+  Renders the `ExploreMap` component, allowing logged-in users to explore the map.  
+  - If the user is not logged in, they will be redirected to `/login`.
```
Comment (SaraPoiatti) [inline code/client/README.md:134]: Guest user can also see the map

File: code/client/README.md [hunk 4]
```diff
@@ -60,82 +99,282 @@ export default tseslint.config({
...
 
-* #### Left Side Bar Component in (`LeftSideBar.tsx`):
-   The component is a sidebar component that provides navigation and user-specific options in the app. It includes a button to create a new document, displays the username, and provides a logout option.
+## 1. General Components
 
-* #### Nav Bar Component in (`NavBar.tsx`):
-   The NavBar component renders a simple navigation bar with a title. It helps maintain a consistent UI by ensuring the navbar is visible across the application, except on the login page.
+### Left Side Bar
+**File**: `components/LeftSideBar.tsx`  
+The component is a sidebar component that provides navigation and user-specific options in the app. It includes a button to create a new document, displays the username, and provides a logout option.
```
Comment (SaraPoiatti) [inline code/client/README.md:193]: menu option for logged in user
-view documents
-new document
-link documents
-documents list

menu option for guest user:
-view documents

File: code/client/README.md [hunk 5]
```diff
@@ -60,82 +99,282 @@ export default tseslint.config({
...
+---
 
-* #### Document Details Component in (`modules/UrbanPlanner/AddDocumentForm/DocumentDetails.tsx`):
-   This component provides input fields for entering a document's title and description. It manages these fields locally and updates the parent component's document state with each change.
+## 2. Anonymous User - Map
 
-* #### Language Selection Component in (`modules/UrbanPlanner/AddDocumentForm/LanguageSelection.tsx`):
-   This component allows users to select a language (either English or Swedish) for a document. It manages the state of the selected language. 
+### ExploreMap 
+**File**: `modules/AnonymousUser-Map/ExploreMap.tsx`
```
Comment (SaraPoiatti) [inline code/client/README.md:224]: This directory has a new name
\modules\Anonymous\Map\ExploreMap

File: code/client/README.md [hunk 6]
```diff
@@ -60,82 +99,282 @@ export default tseslint.config({
...
+---
+   ### Link Type Selection
+   **File**: `modules/UrbanPlanner/AddDocumentForm/elements/LinkTypeSelection.tsx`  
+This component provides a dropdown-based multi-selection interface for choosing link types.
+Enables users to select one or more link types from a predefined list and provides validation feedback if no link type is selected when the form is submitted.
+
+---
+   ### Node Type
+   **File**: `modules/UrbanPlanner/AddDocumentForm/elements/NodeType.tsx`  
+The component is designed to allow users to select the type of a document from a predefined list and updates the document's state with the selected. type.
```
Comment (SaraPoiatti) [inline code/client/README.md:278]: Can you please remove the . between 'selected' and 'type'

File: code/client/README.md [hunk 7]
```diff
@@ -60,82 +99,282 @@ export default tseslint.config({
...
+
+---
+   ### Georeference Type Selection
+   **File**: `modules/UrbanPlanner/AddDocumentForm/elements/Georeference/GeoreferenceTypeSelection.tsx`  
+This component is used for selecting a georeference type for a document.
+
+---
+   ### Mini Map Modal
+   **File**: `modules/UrbanPlanner/AddDocumentForm/elements/Georeference/GeoreferenceTypeSelection.tsx`  
+This component is used for selecting a georeference type for a document.
```
Comment (SaraPoiatti) [inline code/client/README.md:342]: The file is called
modules\UrbanPlanner\AddDocumentForm\elements\Georeference\MiniMapModal.tsx

This component shows a mini map where the user can choose/point a location (lat, long) for the new document

File: code/client/README.md [hunk 8]
```diff
@@ -60,82 +99,282 @@ export default tseslint.config({
...
+### 4.4. Documents List
+**File**: `modules/UrbanPlanner/DocumentsLists/DocumentsListTable.tsx`  
+This component allows users to view, navigate, and interact with document-related actions.
+
+---
+### 4.5. Add resource Form
+**File**: `modules/UrbanPlanner/AddResourceForm/AddResourceForm.tsx`  
+This component is used to upload resources (files) for a specific document. It allows users to select files, view the selected files, and then submit the files.
+
+---
```
Comment (SaraPoiatti) [inline code/client/README.md:378]: you can add here the
\modules\UrbanPlanner\MapFeatures\DraggableMarker.tsx

This components is used by the logged in user to assign a new (lat, long) to the document on the map by moving it with drag&drop

File: code/client/README.md (hunk 5 above)
Comment (SaraPoiatti) [reply code/client/README.md:224]: update:
modules\GeneralPages\Map\ExploreMap.tsx

File: code/client/README.md (hunk 8 above)
Comment (SaraPoiatti) [reply code/client/README.md:378]: update:
\modules\GeneralPages\Map\DraggableMarker.tsx

**Comment by Setareh-Neshatdoust** [comment]
for conflicts issues I close the PR and make new PR in new branch whuch is called Final-Documentation2.

**Pull request closed without merging**